src/crimson/os/seastore/onode_manager/simple-fltree/onode_block.h

```diff
@@ -0,0 +1,61 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <cstdint>
+
+#include "crimson/os/seastore/cached_extent.h"
+#include "onode_delta.h"
+
+namespace crimson::os::seastore {
+
+// TODO s/CachedExtent/LogicalCachedExtent/
+struct OnodeBlock final : CachedExtent {
+  constexpr static segment_off_t SIZE = 4<<10;
```

Comment:
If this is because of page size, it would be useful no explicitly point that out.

Reply:
dropped. as this variable is not used at all.